doc/releases/changelog-dev.md

```diff
@@ -82,20 +156,19 @@
 
   @qml.qnode(dev)
   def circuit():
-
-      generator(wires=range(3))
-      qml.AmplitudeAmplification(U, O, iters=5, fixed_point=True, work_wire=3)
+      generator(wires=range(3)) # prepares |Psi> = U|0>
+      qml.AmplitudeAmplification(U, O, iters=10)
 
       return qml.probs(wires=range(3))
-
   ```
   
   ```pycon
   >>> print(np.round(circuit(), 3))
-  [0.013, 0.013, 0.91, 0.013, 0.013, 0.013, 0.013, 0.013]
-
+  [0.01  0.01  0.931 0.01  0.01  0.01  0.01  0.01 ]
   ```
 
+  As expected, we amplify the $\vert 2 \rangle$ state.
```

Comment:
This does make me wonder "so where do I use `qml.Reflection"?

Reply:
Interesting, I had the opposite thought 😆. @DSGuala you wanna be the tie-breaker?

Reply:
Opposite thought in the sense that it made you completely understand where to use `qml.Reflection`?

Reply:
I think:
- `qml.Reflection` going *first* makes sense if we are building up to `AmplitudeAmplification`
- `qml.Reflection` going *second* makes sense if we are sharing the more important feature first (i.e `AmplitudeAmplification`)

Personally I prefer `qml.Reflection` second. With a comment like: "it's used in the implementation of `AmplitudeAmplification` above"